generator/transit_generator.cpp

```diff
@@ -275,50 +275,89 @@ void GraphData::DeserializeFromJson(my::Json const & root, OsmIdToFeatureIdsMap
   Visit(deserializer);
 }
 
-void GraphData::SerializeToMwm(string const & mwmPath) const
+void GraphData::SerializeToMwm(Writer & writer) const
```

Comment:
The name of the method is wrong now. It serializes to writer, which can be an arbitrary writer.

Reply:
Yes. Let's call it just `Serialize(Writer & writer)`.